test/e2e/framework/ingress_utils.go

```diff
@@ -62,7 +62,12 @@ const (
 	validFor = 365 * 24 * time.Hour
 
 	// Ingress class annotation defined in ingress repository.
-	ingressClass = "kubernetes.io/ingress.class"
+	// TODO: All these annotations should be reused from
+	// ingress-gce/pkg/annotations instead of duplicating them here.
```

Comment:
It is Ok to duplicate the constants in tests? I agree it is a double-edged sword. But the edge pointing away from us is it tells us when someone changes this constant intentionally or unintentionally in the Ingress repo.